addon/-private/formatters/format-time.ts

```diff
@@ -9,6 +9,8 @@ import FormatDateFormatter from './format-date';
  * @private
  * @hide
  */
+// @ts-ignore We could use a proper intermediary abstract class, but it's
+// unnecessarily complicated.
 export default class FormatTime extends FormatDateFormatter {
-  static type = 'time';
+  static readonly type = 'time';
```

Comment:
Redefining `type` is not allowed, as it is `readonly`. If we want to avoid `@ts-ignore` here, we could either remove `readonly`, which would remove some safety, or add another intermediary abstract `DateTimeFormat` base class.

In the long run, it would probably be more sensible to just remove the distinction between `time` and `date` formats, as they both use the same formatter anyway.

Reply:
Ya I can see in the future time and date being merged together.  The main benefit is the separation of the named formats and consistency between [react-intl](https://formatjs.io/docs/react-intl/components#formattedtime) and ember-intl.